src/directives/formly-form.test.js

```diff
@@ -280,6 +280,48 @@ describe('formly-form', () => {
       })
     })
 
+    it(`should be possible to use a wrapper & templateOptions in a fieldGroup`, () => {
+      formlyConfig.setWrapper({
+        name: 'panel',
+        template: `<div class="panel">
+            <div class="heading">
+              Panel Title: {{options.templateOptions.title}}
```

Comment:
Do we have access to the shortcuts when we set wrappers for field groups? I tried accessing the to variable and options.templateOptions is only the way to go.